packages/forms/src/utils/getValidationHue.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import type { Validation } from '../types';
+
+export function getValidationHue(validation?: Validation, defaultHue = 'primaryHue') {
+  switch (validation) {
+    case 'success':
+      return 'successHue';
+    case 'warning':
+      return 'warningHue';
+    case 'error':
+      return 'dangerHue';
+    default:
+      return defaultHue;
+  }
+}
```

Comment:
```suggestion
const VALIDATION_HUES = {
  success: 'successHue',
  warning: 'warningHue',
  error: 'dangerHue'
}

function getValidationHue(validation?: Validation, defaultHue = 'primaryHue') {
  if (validation) {
    return VALIDATION_HUES[validation];
  }
  
  return defaultHue;
  }
}
```

Since this is only used by StyledFauxInput, I would recommend moving it locally to that file.